### DISCUSSION TITLE
Sonraki bölüme geçme tuşu

### DISCUSSION THREAD

**Discussion by KaanAlper** [KebabLord/turkanime-indirici#35] [Genel]
One Piece izliyoruz mesela 700 bölüm izledik, tekrar aratıp 701 e kadar gelmek gerekiyor. Bölümü izledikten sonra sonrakine geçme tuşu gibi bişe eklense olur mu acaba @KebabLord  @gokaybiz

**Comment by KebabLord** [comment]
vaktimi aldı ama buna deydi. Artık seçim ekranı son izlenilen bölümden başlayacak. Yeni sürümü exe olarak da derledim.
[video](https://user-images.githubusercontent.com/24779257/209420172-630e3e8d-a592-42e6-86ba-bd2b57a6408d.webm)

aynı başlıktan bir adet daha olduğu için (#30 ) bunu kilitledim
